src/main/java/be/app/Todo/service/TextFileService.java

```diff
@@ -0,0 +1,18 @@
+package be.app.Todo.service;
+
+import org.springframework.stereotype.Service;
+
+import java.io.FileWriter;
+import java.io.IOException;
+@Service
+public class TextFileService {
+    private static final String FILE_PATH = "errorMessages.txt";
+
+    public void saveMessage(String message) {
```

Comment:
you should use a logging library to do the logging
Log back or slf4j or log4j or sprinboot logging etc etc. They automaticallt log to log files when you do log.info() or log.error()